Cargo.lock

```diff
@@ -1427,7 +1427,7 @@ dependencies = [
 
 [[package]]
 name = "http"
-version = "0.1.18"
+version = "0.1.21"
```

Comment:
I double-checked this new version is actually not vulnerable anymore: https://github.com/hyperium/http/compare/v0.1.18...v0.1.21